dts/arm/nxp/nxp_rt1010.dtsi

```diff
@@ -68,6 +68,10 @@
 	interrupts = <38 0>;
 };
 
+&edma0 {
+	dma-channels = <16>;
+};
+
 / {
```

Comment:
Can this move to a separate commit with details on why this change is needed.

Reply:
Done, the RT1010 has only 16 channels on its EDMA while other RT10xx platforms have 32.